webapp/src/containers/Rhs/index.tsx

```diff
@@ -1,24 +1,49 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 
 import usePluginApi from 'hooks/usePluginApi';
 import {getprojectDetailsState, getRhsState} from 'selectors';
 
+import LinearLoader from 'components/loader/linear';
+
+import plugin_constants from 'plugin_constants';
+
+import AccountNotLinked from './accountNotLinked';
 import ProjectList from './projectList';
 import ProjectDetails from './projectDetails';
 
 const Rhs = (): JSX.Element => {
     const usePlugin = usePluginApi();
 
+    // Fetch the connected account details when RHS is opened
+    useEffect(() => {
+        if (getRhsState(usePlugin.state).isSidebarOpen) {
+            usePlugin.makeApiRequest(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName);
+        }
+    }, []);
+
     if (!getRhsState(usePlugin.state).isSidebarOpen) {
```

Comment:
Same as above. This container won't be rendered if the `sidebar` is not in opened state.